infra/production/base/module/network/main.tf

```diff
@@ -0,0 +1,93 @@
+data "aws_availability_zones" "available" { state = "available" }
```

Comment:
TLDR: Define VPC with private and public subnet topology and use NAT Instance from open source solution